editor/input_event_configuration_dialog.cpp

```diff
@@ -685,9 +685,9 @@ InputEventConfigurationDialog::InputEventConfigurationDialog() {
 	// Key Mode Selection
 
 	key_mode = memnew(OptionButton);
-	key_mode->add_item("Keycode (Latin equvialent)", KEYMODE_KEYCODE);
-	key_mode->add_item("Physical Keycode (poistion of US QWERTY keyboard)", KEYMODE_PHY_KEYCODE);
-	key_mode->add_item("Unicode (case-insencetive)", KEYMODE_UNICODE);
+	key_mode->add_item("Keycode (Latin Equvialent)", KEYMODE_KEYCODE);
+	key_mode->add_item("Physical Keycode (Position of US QWERTY Keyboard)", KEYMODE_PHY_KEYCODE);
+	key_mode->add_item("Unicode (Case-Insensitive)", KEYMODE_UNICODE);
```

Comment:
This should be renamed to match the rest of code.

```suggestion
	key_mode->add_item("Key Label (Unicode, Case-Insensitive)", KEYMODE_UNICODE);
```

Reply:
All these should use TTR probably?